shaky/src/main/java/com/linkedin/android/shaky/DrawFragment.java

```diff
@@ -79,7 +81,17 @@ public void onViewCreated(View view, Bundle savedInstanceState) {
         paper = (Paper) view.findViewById(R.id.shaky_paper);
         imageUri = getArguments().getParcelable(KEY_IMAGE_URI);
         if (imageUri != null) {
-            paper.setImageURI(imageUri);
+            try {
+                // There seems to be an issue when using setImageUri that causes density to be chosen incorrectly
```

Comment:
Add a link to the AOSP bug report: https://code.google.com/p/android/issues/detail?id=201491